tests/old_functional/test_direct_edit.py

```diff
@@ -274,6 +274,24 @@ def test_forbidden_upload(self):
                     == b"Initial content."
                 )
 
+    def test_direct_edit_proxy(self):
+        """
+        Trying to Direct Edit a proxy is not allowed.
+        In that case, the file edition must be aborted an a notification must be shown.
+        """
+        filename = "proxy-test.txt"
+        doc_id = self.remote.make_file("/", filename, content=b"Plein de clics.")
+        folder_uid = self.remote.make_folder("/", "proxy_folder")
+        folder_info = self.remote.get_info(folder_uid)
+        proxy_file = self.remote.create_proxy(doc_id, folder_info.path)
+
+        assert proxy_file["isProxy"] is True
+
+        with patch.object(self.manager_1, "open_local_file", new=open_local_file):
```

Comment:
Even better to be sure nothing is done: you could add a custom `open_local_file()` that would raise a `RuntimeError`.